src/platform/plugins/shared/chart_expressions/expression_xy/public/components/tooltip/tooltip.tsx

```diff
@@ -51,6 +51,38 @@ export const Tooltip: FC<Props> = ({
 }) => {
   const pickedValue = values.find(({ isHighlighted }) => isHighlighted);
 
+  const euiThemeContext = useEuiTheme();
+  const { euiTheme } = euiThemeContext;
+
+  const customToolTipStyles = css`
+    ${euiShadow(euiThemeContext)}
+    ${euiFontSize(euiThemeContext, 's')}
+    padding: ${euiTheme.size.s};
+    z-index: ${euiTheme.levels.toast};
+    border-radius: ${euiTheme.border.radius.medium};
+    background-color: ${euiTheme.colors.backgroundBasePlain};
```

Comment:
Maybe a border could help here for the dark theme:
<img width="386" alt="Screenshot 2025-03-28 at 11 00 32" src="https://github.com/user-attachments/assets/ed2a6174-6b73-42bf-989a-ba1436a5bbd5" />

Reply:
<img width="338" alt="Screenshot 2025-03-28 at 11 25 23" src="https://github.com/user-attachments/assets/5ea3fcea-5012-4da9-93f0-e0af53c07711" />

We don't have borders in general on tooltip. I will check again and find the right color to use instead (that looks a bit darker then the one I selected here

Reply:
Good catch, @dej611. For these sorts of dark-on-dark elements, Borealis utilizes a "floating border". In short, it's a light border that appears only in dark mode. I guess the tooltip mixin doesn't include the floating border styles.

In any case, this is the style you'll likely want to copy from EUI to get that floating border:

```
border: ${euiTheme.border.width.thin} solid ${euiTheme.components.tooltipBorderFloating};
```